### ISSUE TITLE
How to delete an existing project or edit and reload it via DICE

### ISSUE THREAD

**Issue by Piluri** [JRCSTU/CO2MPAS-TA#14]
After I run a CO2MPAS Simulation in TA Mode, I become the Output files. Then I go to DICE and make a new Project for example: IP-10-AAA-2017-1009 and email this project to the stamp server. 
In this time I notice that same data on CO2MPAS Input file are incorrect. After I corrected all this Data and I runed the CO2MPAS simulation in TA Mode again, 
I go to DICE again make a new project and I receive the message that project IP-10-AAA-2017-1009 already exist. 

What should I do at this case? 
The project that exist has the old data. I Need the second one with the correct data. 
Is it possible to delete the old project or edit it and make a new one with the same name?

**Comment by ankostis** [comment]
> Is it possible to delete the old project or edit it and make a new one with the same name?

No, from a legislative point of view, you can never "undo" a dice.

You may edit the Vehicle-Family-ID (e.g. increment the last number by one), 
create *a new project*, and then send a **2nd dice** for the same vehicle,
but that wouldn't make sense if 1st dice was `'sample'`!

You see, your old dice-decision is "valid" for all legal purposes, 
regardless of the correctness of the underlying data.
That is, if the old decision was to *'sample'*, then the OEM will be checked against the old data - and not the 2nd "corrected" one.

__There should be no exception to this rule__, otherwise one could always claim an "error" in co2mpas inputs, in order to avoid sampling checks.

That is why it's important to **ensure that your data are correct BEFORE the `project tsend` command is run.**
The command `project report` has been crafted exactly with this purpose in mind.

Of course, for testing purposes, you may use the test-key as you `master_key`, and do all kinds of experiments.

**Comment by lueginger** [comment]
Dear all, without knowing the details that answer seems not appropriate. No one is asking for changing the decision on that family, exactly for the purpose of keeping in line with the regulation. But never having any typo or similar in one of the millions of data in type approval is not realistic, we are all human beings. That is also reflected in the regulation, that says, if the error identified is not to the benefit of the OEM, everything is OK. So reparing an unintended error should always be possible, as it is in type approval today. So the answer could be, that the dice is left as it is, and the data can be corrected, but dice is not repeated, the decision remains. Is that the solution? Best regards, Christoph Lueginger

**Comment by Piluri** [comment]
At this point I agree with Christoph. 
I think it should be possible, to correct the mistake, re-run the co2mpas simulation (with the corrected Data) and then not to proceed with DICE but to keep the “old” DICE decision. Is that aloud? 

And a second question: where is the last step on Correlation process to make a correction of Data if it’s necessary?

**Comment by MaikeruJon** [comment]
It is true that section 3.2.8 of 2017/1153, as amended by 2017/1213 does say:
"Where the type-approval authority finds that the physical test results do not confirm the input data provided by the manufacturer and, in particular, the data referred to in points 20, 22 and 44 of Table 1 in point 2.4, a verification factor shall be set to 1 and be recorded in the type-approval certificate and in the certificate of conformity. Where the input data is confirmed or where the error in the input data is not to the benefit of the manufacturer the verification factor shall be set to 0. "
However, I don't think that 'error' refers to a mistake. IMHO, this refers to inappropriate selections or value inputs.
However. I also agree that it is not realistic to say that there can never be any mistake.
What are the consequences of the verification factor being '1' or '0'?
Piluri's proposal makes more sense since 2017/1231, which says that **all** CO2MPAS runs must have a DICE run:
"Where the original correlation output report has been issued in accordance with point 3.1.1.1, the type
approval authority, or where applicable the designated technical service shall use the relevant commands
in the correlation tool to send the summary file to a time stamp-server from which a time stamped reply
is returned to the sender (with the relevant services of the Commission in copy), including a randomly
generated integer number between 1 and 99"
Compare that with the original text of 2017/1513:
"Where the NEDC CO 2 value for vehicle H or L is determined in accordance with point 3.2.1, the type-approval authority or the designated technical service shall use the relevant commands in the correlation tool to send the signed summary text file to a time-stamp-server and the following functional mailbox: EC-CO2-LDV-IMPLEMENTATION@ec.europa.eu A time-stamped reply shall be sent in return including a randomly generated integer number in the range 1 to 100 calculated by the correlation tool"
So it seems reasonable to accept the DICE decision and re-run the CO2MPAS with the corrected file.
